hack/specv2-discuss.yaml

```diff
@@ -0,0 +1,213 @@
+      cpu: 50%
+      memory: 70%
+      kafka:
+        - topic: mytopic
+          group: losGroupos
+          maxLag: 123
+  leaderElection:
+    enabled: true
+  secureLogs:
+    enabled: false
+  caBundle:
+    enabled: false
+  # Communication
+  ingress:
+    routes:
+      - host: "myapp.nav.no"
+        path: "/asd"
+        port: 8080 # container port
+        type: http # default
+      - host: "grpc.nav.no"
+        path: "/service"
+        port: 8082
+        type: grpc
+      - host: "myapp-admin.nav.no"
+        path: "/"
+        port: 8081
+    networkAccess:
+      - app: app1 # required
+        namespace: team1
+        ports:
+          - 8081
+    tokenAccess:
+      - app: app1 # required
+        namespace: team1
+        cluster: cluster1 # used to allow applications in other clusters to get tokens
+        permissions: # additional roles/scopes to add to tokens app `name` gets when consuming this app
+          roles:
+            - custom-role
+          scopes:
+            - custom-scope
+  egress:
+    routes:
+      - host: "myapp.nav.no"
+        port: 8080
+      - host: "grpc.nav.no"
+        port: 8080
+      - host: "myapp-admin.nav.no"
+        port: 8080
+    networkAccess: # document that routes will always be available if routes are specified, this is for *in-cluster* communication only
+      - app: app1 # required
+        namespace: team1
+        ports:
+          - 8081
+  # Authentication
+  azure:
+    allowAllUsers: true
+    # claims is removed, we'll automatically assign custom claims to all apps
+    # groups controls both group claims and assignment to azure application
+    allowedGroups:
+      - id: 00000000-0000-0000-0000-000000000000
+    enabled: true
+    # replyURLs is reduced to redirectPath. we'll register redirect URLs by appending the path to the application's ingresses
+    redirectPath: "/oauth2/callback"
+    # singlePageApplication, likely not used - can manually provision if needed
+    tenant: nav.no
+  maskinporten:
+    # enabled: true - automatically enable if list of scopes is non-empty
+    scopes:
+      - name: skatt:scope.read
+      # extract to separate MaskinportenScope k8s resource
+      #exposes:
+      #  - allowedIntegrations:
+      #      - maskinporten
+      #    atMaxAge: 30
+      #    consumers:
+      #      - name: KST
+      #        orgno: "123456789"
+      #    enabled: true
+      #    name: scope.read
+      #    product: arbeid
+  tokenx:
+    enabled: true
+  wonderwall:
+    autoLogin: true
+    autoLoginIgnorePaths:
+      - /path
+      - /internal/*
+    enabled: true
+    errorPath: /error
+    provider: azure # enum of azure|idporten|etc...
+    resources:
+      limits:
+        cpu: 250m
+        memory: 256Mi
+      requests:
+        cpu: 20m
+        memory: 32Mi
+  # Persistence
+  influx:
+    instance: my-influx
+  kafka:
+    - pool: nav-dev
+      streams: false
+    - pool: tbd-dev
+      streams: true
+  openSearch:
```

Comment:
Det er interesse for å kunne bruke flere opensearch instanser, så kanskje dette også burde være en liste?
https://nav-it.slack.com/archives/C5KUST8N6/p1686221727962249